File: src/main/java/ru/jpanda/jenkinsci/plugins/telegrambot/telegram/commands/StartCommand.java
package ru.jpanda.jenkinsci.plugins.telegrambot.telegram.commands;

import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
import org.telegram.telegrambots.meta.api.objects.Chat;
import org.telegram.telegrambots.meta.api.objects.User;
import org.telegram.telegrambots.meta.bots.AbsSender;
import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
import org.telegram.telegrambots.meta.logging.BotLogger;

public class StartCommand extends AbstractBotCommand {

    private static final String LOG_TAG = "/start";

    public StartCommand() {
        super("start", "command.start");
    }

    @Override
    public void execute(AbsSender absSender, User user, Chat chat, String[] strings) {
        SendMessage answer = new SendMessage();
        answer.setChatId(chat.getId().toString());
        answer.setText(String.format(botStrings.get("message.start"),chat.getId()));

        try {
            absSender.execute(answer);
        } catch (TelegramApiException e) {
            BotLogger.error(LOG_TAG, e);
        }
    }
}

File: src/main/kotlin/ru/jpanda/jenkinsci/plugins/telegrambot/telegram/commands/StartCommand.kt
package ru.jpanda.jenkinsci.plugins.telegrambot.telegram.commands

import org.telegram.telegrambots.meta.api.methods.send.SendMessage
import org.telegram.telegrambots.meta.api.objects.Chat
import org.telegram.telegrambots.meta.api.objects.Message
import org.telegram.telegrambots.meta.api.objects.User
import org.telegram.telegrambots.meta.bots.AbsSender
import org.telegram.telegrambots.meta.exceptions.TelegramApiException
import org.telegram.telegrambots.meta.logging.BotLogger

class StartCommand: AbstractBotCommand {

    private val LOG_TAG = "/start"

    constructor():super("start", "command.start")

    override fun execute(absSender: AbsSender, user: User, chat: Chat, strings: Array<String>) {
        val answer = SendMessage()
        answer.chatId = chat.id!!.toString()
        answer.text = botStrings["message.start"]?.let { String.format(it, chat.id) }

        try {
            absSender.execute<Message, SendMessage>(answer)
        } catch (e: TelegramApiException) {
            BotLogger.error(LOG_TAG, e)
        }

    }
}